src/app/EventPathParams.cpp

```diff
@@ -0,0 +1,45 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <app/EventPathParams.h>
+
+namespace chip {
+namespace app {
+
+CHIP_ERROR EventPathParams::BuildEventPath(EventPathIB::Builder & aBuilder) const
```

Comment:
How about move this to a member function of Builder?

i.e. EventPathIB::Builder::BuildEventPath(EventPathParams...)